CONTRIBUTING.md

```diff
@@ -401,6 +367,266 @@ that is not defined in this list below, it will be reviewed for use after you su
 - tracing
 - windows
 
+### Dashboards
+Dashboards are defined under the `dashboards/` directory and CANNOT be nested, a dashboard named `example-dashboard` would live under `dashboards/example-dashboard`. They're defined using a JSON file and a set of screenshots. The dashboard JSON file is generated by exporting a dashboard from New Relic as JSON and removing/updating a few fields.
+
+To convert an exported dashboard to the correct schema:
+1. Remove the `permissions` field
+2. Set every instance of `accountId` to `0`
+3. Set every instance of `linkedEntityGuids` to `null`
+We provide a handy script to do this for you:
+1. Make sure you have `Node 16` or later and `Yarn`
+2. Go into the `utils` directory: `cd utils`
+3. Install dependencies: `yarn install`
+4. Run `yarn sanitize-dashboard example-dashboard`
+
+An example dashboard configuration looks like this:
+```bash
+# in dashboards/example-dashboard directory
+example-dashboard.json
+example-screenshot1.png
+exmaple-screenshot2.png
+```
+
+#### Dashboard Permissions
+
+- When copying dashboard JSON from the New Relic One platform a `permissions` field is always included in the code. You do not need this in a quickstart's dashboard JSON.
+- However if you wish to import a quickstart's dashboard into New Relic outside of the quickstart installation flow, you will need to include this `permissions` field.
+- Refer to this [documentation](https://docs.newrelic.com/docs/query-your-data/explore-query-data/dashboards/dashboards-charts-import-export-data/) on importing / exporting dashboards from the New Relic One platform.
+
+```json
+{
+  "name": "Dashboard Name",
+  "description": null,
+  "permissions": "PUBLIC_READ_WRITE"
+}
+```
+
+#### Dashboard name uniqueness
+
+- A dashboard's name must be unique. After providing a name in the `dashboard.json` file, you can check if your dashboard's name already exists by running `yarn run check_dashboard_name_uniqueness` from the `utils` directory.
+- this [script](https://github.com/newrelic/newrelic-quickstarts/blob/main/utils/check_dashboard_name_uniqueness.js) will check and notify you of duplicate dashboard names in the repository.
+- As a best practice you should run this script when creating a new dashboard.
+
+#### Dashboard screenshots
+
+- Dashboard images are `optional` but highly recommended to preview the visual functionality of a dashboard.
+- File name should be `dashboard_name01`, `dashboard_name02`, etc
+- Dashboards images should be stored in that dashboard's directory with its JSON file. ex: `dashboards/dashboard_name/dashboard_name01.png`.
+- Must be in `.png`, `.jpg`, or `.jpeg` format
+- Each image file must be less than `4MB` in size
+- There should be no more than `12` dashboard or images per quickstart
+- For best results use aspect ratio: 3:2
```

Comment:
Is this accurate for the current UI?

Reply:
That's a good question! Especially the aspect ratio portion. I'll have to go check the IO site